pkg/feature/manifest.go

```diff
@@ -62,28 +75,30 @@ func (m *manifest) targetPath() string {
 	return fmt.Sprintf("%s%s", m.path[:len(m.path)-len(filepath.Ext(m.path))], ".yaml")
 }
 
-func (m *manifest) processTemplate(data interface{}) error {
+func (m *manifest) processTemplate(fs embed.FS, data interface{}) error {
```

Comment:
Same reasons as above
```suggestion
func (m *manifest) processTemplate(fsys fs.FS, data interface{}) error {
```